src/dynamic_fmt.c

```diff
@@ -404,6 +408,46 @@ char *RemoveHEX(char *output, char *input)
 	return output;
 }
 
+// Add $HEX$ encoding for salts where needed.  For canonicalized pot storage, we encode anything
+// containing unprintables or field separators, but never anything else
```

Comment:
I think we normally use C++ style comments only for commenting things out and somehow also in OpenCL - I generally use the old C style comments in our main code in `src`. But I don't really mind, especially if that was already the style in nearby code?

Reply:
Yeah that (already the style in nearby code) was the reason here. It's a mix really. This functions was mostly just moved from `dynamic_utils.c`